.github/workflows/docs-build.yml

```diff
@@ -14,3 +14,4 @@ jobs:
         uses: laminas/documentation-theme/github-actions/docs@master
         env:
           DOCS_DEPLOY_KEY: ${{ secrets.DOCS_DEPLOY_KEY }}
+          "GITHUB_TOKEN": ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I dont get why this is part of this PR ...

Reply:
It's not there anymore.